pyproject.toml

```diff
@@ -18,7 +18,20 @@ dependencies = [
     "gunicorn",
     "pandas",
     "pinecone-client",
+    "python-dotenv",
```

Comment:
this will allow flask to read tokens etc. from the .env file, without needing to export them to the shell

Reply:
that will also work. i.e. both ways are fine. Passing them in as env variables takes priority, in case you want to override it for a single run, or bash shell